services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.test.tsx

```diff
@@ -291,6 +310,17 @@ describe('Release Dialog', () => {
                 },
             ],
         });
+        const status = testcase.rolloutStatus;
+        if (status !== undefined) {
```

Comment:
nitpick:
ideally, this should be a different test array, to avoid `if` statements in test code.
But since this would be against how we do it in this file with the other tests, I would be fine to keep it as it is now.